기업정보 수정

parangdajavous opened the pull request:
* 기업정보 수정 C / S / R

* CompanyInfo Entity에 update 메서드 작성

* Postman Test 시 정상 update 확인

hahamik committed:
user/join

hahamik committed:
user/join

jh0804 reviewed (changes requested):
컨벤션 관련 피드백 드립니다.

첫번째 회의때 Controller와 Service의 메서드명을 일치시키기로 컨벤션을 정하였습니다.
회원가입 코드에서 개인/기업이 다른 Controller 메서드를 사용하지만 Service의 메서드는 동일한 것을 사용하는 것으로 보이기에 이 경우 Controller의 메서드와 Service의 메서드가 다른 것은 괜찮습니다. 
그러나 유저네임중복체크는 위의 경우에 해당하지 않기 때문에 checkUsernameDuplicate -> checkUsernameAvailable로 수정하셔야 될 것 같습니다.

hahamik committed:
user/join

jh0804 committed:
[feat] 회원가입

개인/기업 회원가입 로직 완료
jpa 테스트 코드 작성 완료

hahamik committed:
[feat] 회원가입 (#1)

* 개인/기업 회원가입 C/S/R 완성
* 유저네임 중복체크 완료
* 회원가입 관련 JPA의 테스트 코드 작성 완료

hahamik committed:
Merge branch 'master' of https://github.com/human-kjh/leapit-v2

hahamik committed:
Merge branch 'master' of https://github.com/human-kjh/leapit-v2

hahamik committed:
Merge branch 'master' of https://github.com/human-kjh/leapit-v2

hahamik committed:
Merge branch 'master' of https://github.com/human-kjh/leapit-v2

hahamik committed:
Merge branch 'master' of https://github.com/human-kjh/leapit-v2

hahamik committed:
Merge branch 'master' of https://github.com/human-kjh/leapit-v2

Commit:
companyinfo/update

parangdajavous committed:
[feat] 기업정보 수정 (#11)

기업정보 수정 C / S / R

CompanyInfo Entity에 update 메서드 작성

Postman Test 시 정상 update 확인

Commit:
기업정보 수정 완료

Commit:
충ë돌해결
